src/utilities/device_utils.c

```diff
@@ -416,17 +416,9 @@ hypre_GetDefaultDeviceGridDimension( HYPRE_Int n,
 }
 
 __global__ void
-hypreGPUKernel_IVAXPY(
```

Comment:
So, all (CUDA) kernels will have `hypre_Item &item` which is `void *` as the first arg. in the future? It's OK to waste a register space I guess.